usr/share/rear/conf/default.conf

```diff
@@ -949,3 +949,5 @@ export TMPDIR    # the export is required so that mktemp can pickup the variable
 # Specify if rear is managed from DRLM (y/n) [ default (n) ].
 DRLM_MANAGED=n
 
+# Specify if rear should try to backup capabilities (y/n) default (n).
+BACKUP_CAP=n
```

Comment:
Since the code for BACKUP_CAP is only used for BACKUP=NETFS please rename the variable to something like `NETFS_RESTORE_CAPABILITIES`. That way it will show up if you run `rear dump` and the name of the variable is much more self expalantory.

Also, please move this variable to the NETFS section in `default.conf`